tests/element_test.py

```diff
@@ -1,12 +1,13 @@
 u"""Tests for Crystal and CrystalSlice
 """
 from ast import Assert
+from re import L
 from pykern.pkdebug import pkdp, pkdlog
 from pykern.pkcollections import PKDict
+from pykern.pkunit import pkexcept
```

Comment:
Import pykern.pkunit. The only special cases are pkdebug and pkcollections. Even those are probably a mistake...